Scripts/Utilities/ObjectUtils.cs

```diff
@@ -278,9 +278,26 @@ public static string NicifySerializedPropertyType(string type)
 		/// <returns>Best guess System.Type</returns>
 		public static Type TypeNameToType(string name)
 		{
-			return AppDomain.CurrentDomain.GetAssemblies()
-				.SelectMany(x => x.GetTypes())
-				.FirstOrDefault(x => x.Name.Equals(name) && typeof(UnityObject).IsAssignableFrom(x));
+			var assemblies = AppDomain.CurrentDomain.GetAssemblies();
+			foreach (var assembly in assemblies)
+			{
+				try
+				{
+					foreach (var type in assembly.GetTypes())
+					{
+						if (type.Name.Equals(name) && typeof(UnityObject).IsAssignableFrom(type))
+						{
+							return type;
+						}
```

Comment:
remove brackets on single line if

Reply:
Not quite sure why the inner part was de-linqified

Reply:
Because exception handling is now needed